daml-script/test/daml/upgrades/UpgradesTest.daml

```diff
@@ -59,13 +61,21 @@ main = forA_ tests $ \(testName, test) -> do
 brokenScript : Script () -> Script ()
 brokenScript act = do
   tryToEither (\() -> liftFailedCommandToException act) >>= \case
-    Right _ -> assertFail "Expected failed and got success! Did you fix this logic? Remove the wrapping `failure` to mark this as working."
+    Right _ -> assertFail "Expected failed and got success! Did you fix this logic? Remove the wrapping `broken` to mark this as working."
     Left _ -> do
       -- Cleanup so next test can run.
-      vetDarOnParticipant v1DarName participant0
-      vetDarOnParticipant v1DarName participant1
-      vetDarOnParticipant v2DarName participant0
-      vetDarOnParticipant v2DarName participant1
+      setProvidePackageId False
+      -- Unvet first so we don't end up vetting a package twice
+      unsafeUnvetDarOnParticipant v1DarName $ Some participant0
+      unsafeUnvetDarOnParticipant v1DarName $ Some participant1
+      unsafeUnvetDarOnParticipant v2DarName $ Some participant0
+      unsafeUnvetDarOnParticipant v2DarName $ Some participant1
+      sleep $ seconds 1
+      unsafeVetDarOnParticipant v1DarName $ Some participant0
+      unsafeVetDarOnParticipant v1DarName $ Some participant1
+      unsafeVetDarOnParticipant v2DarName $ Some participant0
+      unsafeVetDarOnParticipant v2DarName $ Some participant1
+      sleep $ seconds 1
```

Comment:
Improved cleanup using unsafe variant, so we only delay 2s, rather than 4s previously for `broken` tests